quickstart.py

```diff
@@ -551,10 +550,10 @@ def print_results(request_id: str) -> None:
     input()
 
     create_connection(
-        key="test-application-postgres-db", connection_type=ConnectionType.postgres
+        key="test_application_postgres_db", connection_type=ConnectionType.postgres
```

Comment:
Could you similarly update the [README.md](https://github.com/ethyca/fidesops#rocket-quick-start) that has the instructions for this quickstart, so the keys are snake cased?